src/code/audio/code_8019AF00.c

```diff
@@ -217,768 +236,893 @@ s8 gUnderwaterSfxReverbAdd = 0;
+    SEQ_FLAG_FANFARE,                     // NA_BGM_BALLAD_OF_THE_WIND_FISH
+    SEQ_FLAG_RESTORE,                     // NA_BGM_SONG_OF_SOARING
+    0,                                    // NA_BGM_MILK_BAR_DUPLICATE
+    0,                                    // NA_BGM_FINAL_HOURS
+    SEQ_FLAG_FANFARE,                     // NA_BGM_MIKAU_RIFF
+    SEQ_FLAG_FANFARE,                     // NA_BGM_MIKAU_FINALE
+    0,                                    // NA_BGM_FROG_SONG
+    SEQ_FLAG_FANFARE,                     // NA_BGM_OCA_SONATA
+    SEQ_FLAG_FANFARE,                     // NA_BGM_OCA_LULLABY
+    SEQ_FLAG_FANFARE,                     // NA_BGM_OCA_NEW_WAVE
+    SEQ_FLAG_FANFARE,                     // NA_BGM_OCA_ELEGY
+    SEQ_FLAG_FANFARE,                     // NA_BGM_OCA_OATH
+    0,                                    // NA_BGM_MAJORAS_LAIR
+    SEQ_FLAG_FANFARE,                     // NA_BGM_OCA_LULLABY_INTRO
+    SEQ_FLAG_FANFARE,                     // NA_BGM_OCA_GUITAR_BASS_SESSION
+    SEQ_FLAG_FANFARE,                     // NA_BGM_PIANO_SESSION
+    SEQ_FLAG_FANFARE,                     // NA_BGM_INDIGO_GO_SESSION
+    SEQ_FLAG_ENEMY,                       // NA_BGM_SNOWHEAD_TEMPLE
+    SEQ_FLAG_ENEMY,                       // NA_BGM_GREAT_BAY_TEMPLE
+    SEQ_FLAG_FANFARE,                     // NA_BGM_NEW_WAVE_SAXOPHONE
+    SEQ_FLAG_FANFARE,                     // NA_BGM_NEW_WAVE_VOCAL
+    0,                                    // NA_BGM_MAJORAS_WRATH
+    0,                                    // NA_BGM_MAJORAS_INCARNATION
+    0,                                    // NA_BGM_MAJORAS_MASK
+    SEQ_FLAG_FANFARE,                     // NA_BGM_BASS_PLAY
+    SEQ_FLAG_FANFARE,                     // NA_BGM_DRUMS_PLAY
+    SEQ_FLAG_FANFARE,                     // NA_BGM_PIANO_PLAY
+    SEQ_FLAG_ENEMY,                       // NA_BGM_IKANA_CASTLE
+    0,                                    // NA_BGM_GATHERING_GIANTS
+    SEQ_FLAG_FANFARE_KAMARO,              // NA_BGM_KAMARO_DANCE
+    0,                                    // NA_BGM_CREMIA_CARRIAGE
+    SEQ_FLAG_FANFARE,                     // NA_BGM_KEATON_QUIZ
+    0,                                    // NA_BGM_END_CREDITS_1
+    0,                                    // NA_BGM_OPENING_LOOP
+    0,                                    // NA_BGM_TITLE_THEME
+    SEQ_FLAG_FANFARE,                     // NA_BGM_DUNGEON_APPEAR
+    SEQ_FLAG_FANFARE,                     // NA_BGM_WOODFALL_CLEAR
+    SEQ_FLAG_FANFARE,                     // NA_BGM_SNOWHEAD_CLEAR
+    0,                                    //
+    0,                                    // NA_BGM_INTO_THE_MOON
+    0,                                    // NA_BGM_GOODBYE_GIANT
+    0,                                    // NA_BGM_TATL_AND_TAEL
+    0,                                    // NA_BGM_MOONS_DESTRUCTION
+    0,                                    // NA_BGM_END_CREDITS_2
 };
 
 s8 sSpecReverbs[20] = {
```

Comment:
What is the 20? Is it the same 20 as sAmbienceData?

Reply:
No, it's different. It's `SPEC_ID_MAX` i.e. the total number of specs:
https://github.com/engineer124/mm/blob/8dd2a6ccf505f62e1aa7a40f80c6b218d6fece2f/src/code/audio/audio_init_params.c#L120-L183

... except there's 21 specs and it seems like this was copied from OoT withou expanding properly. Hurray. Probably deserves a @ bug, although I can do that when I import `AudioSpec gAudioSpecs[21]`